extensions/panache/hibernate-orm-panache-kotlin/runtime/src/main/kotlin/io/quarkus/hibernate/orm/panache/kotlin/PanacheEntityBase.kt

```diff
@@ -22,7 +22,9 @@ interface PanacheEntityBase {
      *
      * @return the default [EntityManager]
      */
-    fun entityManager(): EntityManager = JpaOperations.getEntityManager()
+    @JsonbTransient
+    @JsonIgnore
+    fun getEntityManager(): EntityManager = JpaOperations.getEntityManager()
```

Comment:
```suggestion
    fun getEntityManager() = JpaOperations.getEntityManager()
```

I think this should be possible.

Same here https://github.com/quarkusio/quarkus/pull/10611/files#diff-abda2266af87d72cd3c68169ac202d83R25

Reply:
Intellij recommends using the type. If I do that it warns me with `Declaration has type inferred from a platform call, which can lead to unchecked nullability issues. Specify type explicitly as nullable or non-nullable`